powergenome/renewables_clusters.py

```diff
@@ -320,17 +329,21 @@ def build_clusters(
         raise ValueError("Max number of clusters must be greater than zero")
     df = metadata
     cdf = _get_base_clusters(df, ipm_regions)
-    if cdf.empty:
-        raise ValueError(f"No resources found in {ipm_regions}")
     cdf = cdf.sort_values("lcoe")
     if min_capacity:
         # Drop clusters with highest LCOE until min_capacity reached
-        end = cdf[CAPACITY].cumsum().searchsorted(min_capacity) + 1
-        if end > len(cdf):
-            raise ValueError(
-                f"Capacity in {ipm_regions} ({cdf[CAPACITY].sum()} MW) less than minimum ({min_capacity} MW)"
-            )
+        end = min(len(cdf), cdf[CAPACITY].cumsum().searchsorted(min_capacity) + 1)
         cdf = cdf[:end]
+    if max_lcoe:
+        # Drop clusters with LCOE above the cutoff
+        cdf = cdf[cdf["lcoe"] > max_lcoe]
```

Comment:
I think this should be `cdf = cdf[cdf["lcoe"] <= max_lcoe]`

Reply:
Ahem, yes, good catch. Fixed in 3df122e8c2267083ba7bbd09ba4537202960668e.